src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -250,9 +250,9 @@ private void testAddAction() throws Exception {
         feedbackPage.clickNeverVisibleTimeButton();
 
         //verify that timeFrameTable, instructions and ResponseVisTable are all hidden
-        feedbackPage.verifyHidden(By.id("timeFramePanel"));
-        feedbackPage.verifyHidden(By.id("responsesVisibleFromColumn"));
-        feedbackPage.verifyHidden(By.id("instructionsRow"));
+        assertTrue(feedbackPage.verifyHidden(By.id("timeFramePanel")));
```

Comment:
@damithc Initially i thought it would be a little bigger issue before starting so took it but later realized  its too small.
I will take care of this in future :sweat_smile: